test/suites/launchTemplates/suite_test.go

```diff
@@ -0,0 +1,93 @@
+package launchtemplates
```

Comment:
Thoughts on keeping this in the integration suite? Right now suites are parallel, but pay a large startup cost. Unless this relies on addtl dependencies (CSI, etc), maybe best to keep them together.

Reply:
I can go either way. I expect this entire suite of tests to call EC2 + SSM to verify that the actual configuration on the EC2 instances matches our requirements in the provisioner, but shouldn't be other dependencies.

Reply:
Cool. If no other deps, let's keep it in the integ suite.

Reply:
Done!